telegram/error.py

```diff
@@ -100,3 +99,12 @@ def __init__(self, retry_after):
         super(RetryAfter,
               self).__init__('Flood control exceeded. Retry in {} seconds'.format(retry_after))
         self.retry_after = float(retry_after)
+
+
+class Conflict(TelegramError):
+    def __init__(self, msg, url):
+        match = re.search(r'bot(\d+):.*/', url)
```

Comment:
Can you please add a comment on the code itself  what is the input string and why it means a conflicting bot id? (in favour of future us who won't remember all the details...)